### ISSUE TITLE
Errors due to new version of NerfAcc

### ISSUE THREAD

**Issue by IHe-KaiI** [autonomousvision/sdfstudio#83]
Hi @liruilong940607 and all! 

Thanks for the wonderful work all your team!

I found when I run `ns-install-cli`, it has some errors (eg. `ImportError: cannot import name 'OccupancyGrid' from 'nerfacc'`). It may occur due to the newest version of NerfAcc, and I can run smoothly with NerfAcc-0.3.5.

Best, 
Kai

**Comment by liruilong940607** [comment]
Yeah it would require some updates to work with the latest nerfacc. It should specify nerfac c <=0.3.5
